packages/eslint-plugin-i18n/src/rules/interpolation-data.js

```diff
@@ -35,7 +35,16 @@ module.exports = {
           return;
         }
 
-        const [keyNode, dataNode, countNode] = node.arguments;
+        const [keyNode] = node.arguments;
+        let [, dataNode, countNode] = node.arguments;
```

Comment:
It's ok to use a let for a value (even if it should be a const) if the linter let you do so 🤔 
```suggestion
        let [keyNode, dataNode, countNode] = node.arguments;
```

Reply:
Yes, lint issue... :sadpanda: